src/pages/ct-default/form/CtDefaults.js

```diff
@@ -39,24 +28,27 @@ const CtDefaults = ({ _labels, access }) => {
     'ct_credit_sales_ratetype_id',
     'ct_credit_purchase_ratetype_id',
     'ct_credit_eval_ratetype_id',
-    'ct_minOtp_CIVAmount'
+    'ct_minOtp_CIVAmount',
+    'otp-expiry-time'
   ]
-
   useEffect(() => {
     getDataResult()
-    getData()
   }, [access])
 
   const formik = useFormik({
     enableReinitialize: true,
     validateOnChange: true,
+    validationSchema: yup.object({
+      'otp-expiry-time': yup.number().min(30).max(120)
+    }),
     initialValues: {
       'ct-nra-individual': null,
-      'ct-nra-individual-ref': null,
-      'ct-nra-individual-description': null,
+      'ct-nra-individual-ref': '',
```

Comment:
Only include the fields required by api in formik, remove:
      'ct-nra-corporate-ref': '',
      'ct-nra-corporate-description': '',
       'ct-nra-individual-ref': '',
      'ct-nra-individual-description': '',